docs/adr/template.md

```diff
@@ -68,5 +68,5 @@ Chosen option: "[option 1]", because [justification. e.g., only option, which me
 
 ## Links <!-- optional -->
 
-* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
+* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0000](0000-logging-warnings-using-the-python-warnings-or-logging-module.md) -->
```

Comment:
The same issue also exists in the status field.

Reply:
indeed I didn't see, thanks

Reply:
The link-check tests don't run by default on Jenkins, so you have to start them manually. See: https://jenkins.inmanta.com/job/docs/job/inmanta-core-docs-test/job/fix-link-tempalte-adr